### PR TITLE
Change definition of "path" in Okanshi.WebApi

### LINKED ISSUE #147: fix "memory leak" from webapi

### PR DESCRIPTION
**Pull request by kbilsted**
close #147

### CODE REVIEW

**Comment by kichristensen** [comment]
I think you should be able to opt in for that functionality, as both is valid

File: src/Okanshi.WebApi/OkanshiWebApiOptions.cs [hunk 1]
```diff
@@ -17,9 +17,33 @@ public class OkanshiWebApiOptions
...
 		/// </summary>
 		public Func<string, Tag[], ITimer> TimerFactory = (name, tags) => OkanshiMonitor.Timer(name, tags);
 	}
+
+	/// <summary>
+    /// How the request path is extracted
+    /// </summary>
+	public enum RequestPathExtraction
+	{
+        /// <summary>
```
Comment (kichristensen) [inline src/Okanshi.WebApi/OkanshiWebApiOptions.cs]: Please fix indention

**Comment by kbilsted** [comment]
Is there a good way to ensure that this uses the evicting registry? perhaps just documentation missing?

**Comment by kichristensen** [comment]
I would say it just is documentation, and leave it to user to decide what is the wanted behaviour

**Comment by kbilsted** [comment]
I'll write some doc then. Unless you want to ;)

**Comment by kbilsted** [comment]
🚢

**Comment by kbilsted** [comment]
rebased

**Comment by kbilsted** [comment]
I think you have write rights so you can rebase without having to wait for me

**Pull request merged**